appveyor.yml

```diff
@@ -0,0 +1,18 @@
+os: Visual Studio 2015
+
+branches:
```

Comment:
branches is not needed since the only branches that get built are ones which have yml file!